module/os/freebsd/zfs/sysctl_os.c

```diff
@@ -679,6 +679,26 @@ param_set_deadman_failmode(SYSCTL_HANDLER_ARGS)
 	return (-param_set_deadman_failmode_common(buf));
 }
 
+int
+param_set_raidz_impl(SYSCTL_HANDLER_ARGS)
+{
+	char *buf;
+	int rc;
+
+	buf = malloc(64, M_SOLARIS, M_NOWAIT | M_ZERO);
```

Comment:
May be we could give some more space here just in case.  64 bytes we could allocate even on stack.  But whatever.